src/main/java/com/uid2/shared/store/FileName.java

```diff
@@ -0,0 +1,24 @@
+package com.uid2.shared.store;
+
+public class FileName {
+    private final String prefix;
+    private final String suffix;
```

Comment:
Does suffix here refer to the filename itself? Or is prefix the filename and suffix the file extension?

Reply:
Yes, suffix is the extension including the dot.

Reply:
Could you add a comment there specifying that?